content/_data/upgrades/2-277-1.adoc

```diff
@@ -0,0 +1,51 @@
+==== Configuration Form Modernization
+
+The transition from HTML `table` layout to HTML `div` layout in configuration forms requires changes in some plugins for compatibility.
+Jenkins administrators should update the installed plugins when they upgrade to Jenkins 2.277.1.
+Some older plugin releases will cause the Jenkins "Configure System" page to fail to save.
+Other older plugin releases will report Java runtime exceptions until they are updated to the most recent release.
+
+Use the Plugin Manager to update plugins immediately after upgrading to Jenkins 2.277.1.
```

Comment:
Well, you should also be checking for updates _before_ upgrading, to pick up fixes for JEP-227/228 incompatibilities. (Which are all available for older Jenkins versions with one exception: `cas-plugin`.)

Reply:
@jglick are there examples of JEP-227 and JEP-228 failure modes from outdated plugins that we could include in those sections?  I've experienced the "Configure System" failure and the runtime exceptions, so was able to use them as examples of the types of failures.

I think those examples would increase the chances of the text being found by search engines, if they are available.

Reply:
Mentioned specific exceptions in 52f7baa09fc37149ed5ba99cd3a1f399404f6b71.  Also included instructions that they should update plugins before they upgrade to 2.277.1.